app/Components/SudokuBoard/Components/Hint.tsx

```diff
@@ -0,0 +1,144 @@
+import { Text, useTheme } from "react-native-paper";
+import { HintObjectProps } from "../SudokuBoard";
+import { formatOneLessonName } from "../../../Functions/learnedLessons";
+import { Platform, Pressable, View } from "react-native";
+import { getCellSize } from "../Functions/BoardFunctions";
+import { MaterialCommunityIcons } from "@expo/vector-icons";
+import React from "react";
+
+interface HintProps extends HintObjectProps {
+  incrementStage: (stageOffset: number) => void;
+}
+
+const Hint = (hintProps: HintProps) => {
+  const { stage, hint, maxStage, incrementStage } = hintProps;
+
+  const cellSize = getCellSize();
+  const theme = useTheme();
+
+  const sizeConst = Platform.OS == "web" ? 1.5 : 1;
+  let FALLBACK_HEIGHT = 30;
+
+  const STRATEGY_FONT_SIZE = 30;
+
+  const hintTitle = (
+    <Text
+      style={{
+        textAlign: "center",
+        fontSize: STRATEGY_FONT_SIZE,
+        marginBottom: 10,
+        color: theme.colors.primary,
+      }}
+    >
+      {formatOneLessonName(hint.strategy)}
+    </Text>
+  );
+
+  interface StageContent {
```

Comment:
A comment or two in "stage" section would be nice especially one explaining what stage means in this context

Reply:
I'm not sure how I would explain it, it is a hint stage

Reply:
Maybe it's worth naming it HintStageContent? Was confused cause when I hear stage content I think of git not steps in a hint.

Reply:
That works for me, I renamed!